js/data.js

```diff
@@ -15,6 +15,16 @@ var data = {
   submittedWords: [],
   submittedCard: 0,
   score: 0,
-  runningScore: 0
+  runningScore: 0,
+  completed: false
+};
+
+const previousPlaylist = localStorage.getItem('javascript-local-storage');
+if (previousPlaylist !== null) {
+  data.playlist = JSON.parse(previousPlaylist);
+}
```

Comment:
Can you incorporate this into your whole data model, so you don't have to read from local storage in more than one place?